megamek/src/megamek/common/AbstractGame.java

```diff
@@ -110,6 +110,7 @@ public synchronized Forces getForces() {
     }
 
     @Override
+    @Nullable
```

Comment:
the original function this overrides is `@Nullable`, so it's important to keep the annotation in the override function, otherwise the IDE may not figure out that a call can return null (which is exactly what was happening).